homeassistant/components/zoneminder.py

```diff
@@ -0,0 +1,119 @@
+"""
+Support for Zoneminder.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/zoneminder/
+"""
+
+import logging
+import json
+from urllib.parse import urljoin
+
+import requests
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.const import (
+    CONF_URL, CONF_HOST, CONF_PASSWORD, CONF_USERNAME)
+
+
+_LOGGER = logging.getLogger(__name__)
+
+REQUIREMENTS = []
+
+DOMAIN = 'zoneminder'
+
+CONFIG_SCHEMA = vol.Schema({
+    DOMAIN: vol.Schema({
+        vol.Required(CONF_HOST): cv.string,
+        vol.Required(CONF_URL): cv.string,
```

Comment:
Its just a php webpage running under apache, so it can really just live anywhere.  I believe /zm/ is the default (at least by the automated installer), so we should set it to be the default and be able to be overrided.  Didn't even realize I missed that part, I'll get it fixed with sane defaults :)